parser/testdata/workflows/dataInputSchemaNotExists.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024 The Serverless Workflow Specification Authors
```

Comment:
is this file being used?

Reply:
Yes, it is the new test case. There's a test case in which we go through each file in this folder and perform a parse.